docs/kylinlinux.md

```diff
@@ -0,0 +1,11 @@
+# Kylin Linux
+
+Kylin Linux is currently only supported with containerd runtime.
+
+**Note:** that Kylin Linux is not currently covered in kubespray CI and
+support for it is currently considered experimental.
```

Comment:
What is a blocker to implement CI for Kylin Linux?

Reply:
😁 I actually don't know much about ci, could you tell me how to do it? @oomichi

Reply:
OK, that is a good question.
I did the similar thing for RockyLinux 8 which is compatible with RHEL8/CentOS8 as https://github.com/kubernetes-sigs/kubespray/pull/8905

The pull request would be helpful to know how to do that.

Reply:
@oomichi @cristicalin @floryut  🤔 Kylin Linux is a commercial operating system that does not provide public vagrant box and qcow2 image files. Is there any way to deal with this situation?

Reply:
IICU the Kubespray's operating system check blocks running on Kylin OS and it is fine to make the check allow running without CI.
That is the same as RHEL, we don't have any RHEL CI job but Kubespray allows running on the RHEL.
If users find an issue, they report and fix it.

@ErikJiang 
Thanks for your work, please revert this pull request to the previous one which changed the check and wrote the document which explained the Kylin support is experimental.

Thanks

Reply:
@oomichi 😁 I have restored the PR to the previous state, if kylin can be accepted by kubespray, I will be very willing to apply to be one of the maintainers of kylin, and I believe that more Chinese developers will contribute to this.

Reply:
@ErikJiang Thank you for doing that.

>  I believe that more Chinese developers will contribute to this.

Yeah, I also guess so.

Reply:
Or https://www.openkylin.top/index.php?lang=en ? 😆 joking, as @oomichi and @cristicalin it's fine to not have CI support and only experimental support as long as people try and fix issues with it (if anyone uses it)

Reply:
At present, `kylin linux V10` is more mature than `openkylin`, so `openkylin` will not be considered in the short term.